src/CentreonRemote/Application/Clapi/CentreonWorker.php

```diff
@@ -118,14 +118,16 @@ public function createRemoteTask(int $taskId)
          */
         $params = unserialize($task->getParams())['params'];
         $centreonPath = trim($params['centreon_path'], '/');
+        $centreonPath = (isset($centreonPath) && strcmp($centreonPath, '') != 0) ? $centreonPath : '/centreon';
```

Comment:
an isset function is not necessary because on row 120 you are defined the variable $centreonPath
```suggestion
        $centreonPath = $centreonPath ? $centreonPath : '/centreon';
```